0From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paper Developer <paper@papermc.io>
Date: $(date)
Subject: [PATCH] Set online-mode default to false

Changes the default value of online-mode in server.properties from true to false
for easier offline development and testing.

diff --git a/src/main/java/net/minecraft/server/dedicated/DedicatedServerProperties.java b/src/main/java/net/minecraft/server/dedicated/DedicatedServerProperties.java
index xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx..yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy 100644
--- a/src/main/java/net/minecraft/server/dedicated/DedicatedServerProperties.java
+++ b/src/main/java/net/minecraft/server/dedicated/DedicatedServerProperties.java
@@ -XX,X +XX,X @@ public class DedicatedServerProperties extends Settings<DedicatedServerPropertie
         this.queryPort = this.get("query.port", this.serverPort);
         this.enableRcon = this.get("enable-rcon", false);
         this.rconPort = this.get("rcon.port", 25575);
-        this.onlineMode = this.get("online-mode", true);
+        this.onlineMode = this.get("online-mode", false); // Paper - Set online-mode default to false
         this.allowFlight = this.get("allow-flight", false);
         this.motd = this.get("motd", "A Minecraft Server");
         this.forceGamemode = this.get("force-gamemode", false);

# by __DEMORA__